faim_robocopy/robocopy.py

```diff
@@ -346,6 +346,11 @@ def _submit(destination):
             # whenever a source and destination have different content.
             while self.is_running():
 
+                # delete files that are copied to all destinations.
```

Comment:
Now that this part runs _before_ the actual robocopy jobs, maybe we can clarify the comment somehow: 

```suggestion
                # delete files that are present in all destinations.
```